local_planning:
  g_zero: 0.005
  num_yaw_samples: 10
  uav_radius: 1.5 # [m]

frame_id: "world_origin"

body:
  frame_id: "fcu"

camera:
  frame_id: "rgbd_front_pitched/aligned_depth_to_color"
  h_fov: 1.51844 # [rad]
  width: 1080
  height: 720
  min_distance: 0.2 # [m]
  max_distance: 5 # [m]

bounded_box:
  min_x: -25.0 # [m]
  max_x: 25.0 # [m]
  min_y: -20.0 # [m]
  max_y: 15.0 # [m]
  min_z: 1.0 # [m]
  max_z: 20.0 # [m]
  spacing: 1.0 # [m]
